modules/clair-postgresql-database-update.adoc

```diff
@@ -0,0 +1,99 @@
+[id="upgrading-clair-postgresql-database"]
+= Upgrading the Clair PostgreSQL database
+
+If you are upgrading your standalone {productname} deployment to version 13, you must migrate your Clair PostgreSQL database version from PostgreSQL version 13 -> version 15. This requires bringing down your Clair PostgreSQL 13 database and running a migration script to initiate the process.
```

Comment:
`... deployment to version 13` -> `... deployment to version 3.13`? hear the version you mean is Quay, right?

Reply:
sorry @stevsmit, I still see previous characters, no update?  
I mean `to version 13` should be `to version 3.13`, if the version you point to Quay.  
13/15 are version for PostgreSQL database. are the version you mean PostgreSQL?
below has another same error, pls also take care